libcloud/common/abiquo.py

```diff
@@ -125,10 +125,16 @@ def parse_error(self):
             raise InvalidCredsError(driver=self.connection.driver)
         elif self.status == httplib.FORBIDDEN:
             raise ForbiddenError(self.connection.driver)
+        elif self.status == httplib.NOT_ACCEPTABLE:
+            raise LibcloudError('Not Acceptable')
         else:
-            errors = self.parse_body().findall('error')
-            # Most of the exceptions only have one error
-            raise LibcloudError(errors[0].findtext('message'))
+            parsebody = self.parse_body()
+            if parsebody is not None and hasattr(parsebody, 'findall'):
+                errors = self.parse_body().findall('error')
+                # Most of the exceptions only have one error
+                raise LibcloudError(errors[0].findtext('message'))
+            else:
+                raise LibcloudError('unparseable body')
```

Comment:
Probably better to at least return raw body in the exception?

Otherwise it will be hard to debug it without using LIBCLOUD_DEBUG.